<launch>
  <node name='follower' pkg="ros_tracker" type="follower_no_map.py">
    <!-- switchMode: (if true one button press will change betwenn active, not active. If false the button will have to be kept pressed all the time to be active -->
    <param name="switchMode" value="True" type="bool" />
    <!-- maximal speed (angular and linear both), tartet distance: the robot will try to keep this fixed distance -->
    <param name='maxSpeed' value='0.4' type='double' />
    <param name='targetDist' value='2.0' type='double' />
    <param name='maxGimbalSpeed' value='1.0' type='double'/>
    <param name='car_radius' value='0.133' type='double'/>
    <!-- index of the button in the buttons field of the joy message from the ps3 controller, that switches active/inactive  -->
    <param name='controllButtonIndex' value='-4' type='int' />
    <rosparam ns='PID_controller' command='load' file='$(find ros_tracker)/parameters/PID_param.yaml' />
    <rosparam ns='dwa_param' command='load' file='$(find ros_tracker)/parameters/dwa_param.yaml'/>
  </node>
</launch>